electronics_lib/Passives.py

```diff
@@ -314,6 +297,224 @@ def derated_capacitance(row: Dict[str, Any]) -> Tuple[float, float]:
       raise ValueError(f"no single capacitors in ({capacitance}) F")
 
 
+class SmtCeramicCapacitorGeneric(Capacitor, FootprintBlock, GeneratorBlock):
+  @init_in_parent
+  def __init__(self, *args, **kwargs):
+    super().__init__(*args, **kwargs)
+
+    self.footprint_spec = self.Parameter(StringExpr(""))
+
+    # Default to be overridden on a per-device basis
+    self.single_cap_max = 22e-6
```

Comment:
Should this be a `self.` field, or is this just a classwide (or even local function) constant?